mappings/net/minecraft/network/ClientConnection.mapping

```diff
@@ -29,6 +29,7 @@ CLASS none/eq net/minecraft/network/ClientConnection
 	FIELD p disconnected Z
 	METHOD <init> (Lnone/fm;)V
 		ARG 0 side
+	METHOD a tick ()V
```

Comment:
I'd say you can keep it as `tick` and we'll rename the bunch in a later PR / commit.